### DISCUSSION TITLE
Webhook WPGive

### DISCUSSION THREAD

**Discussion by DoAgency** [impress-org/givewp#7051] [Developer Q&A] [answered]
Hi,
**1) Legacy Form**
When there is a donation, that triggers webhook always. If donatin is completed and also if donation is pending.
**2) Multistep form**
When there is a donation, that triggers webhook just in case donation is completed.

Is this correct?
If that is so. I have a question:
What are name of status of json response?
"publish" is the status when donation is completed and "pending" if donation is pending?

**Comment by JoshuaHungDinh** [comment, marked as answer]
Hey @DoAgency, here is a complete list of donation statuses from the DonationActions endpoint

``` 
'status' => [
                        'type' => 'string',
                        'required' => false,
                        'enum' => [
                            'publish', // Completed
                            'pending',
                            'processing',
                            'refunded',
                            'revoked',
                            'failed',
                            'cancelled',
                            'abandoned',
                            'preapproval',
                        ],
                    ],
```
